### PR TITLE
Validation: Added Missing Descriptions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing metadata descriptions.

### CODE REVIEW

**Comment by dereklh7** [comment]
@stevemunk @rbrundritt 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: BingMaps/v8-web-control/map-control-concepts/clustering-module-examples/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Clustering Module Examples | Microsoft Docs"
+description: "Describes the Clustering module and provides a list of examples of the Clustering module and a list of additional references."
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/clustering-module-examples/index.md:3]:
```suggestion
description: "Describes the Clustering module with a list of examples and related topics."
```

File: BingMaps/v8-web-control/map-control-concepts/configuration-driven-maps-framework/iframeable-configuration-map-example.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "IFrameable Configuration Map Example | Microsoft Docs"
+description: "Describes the Iframeable configuration map and provides an example with HTML and provides an example to try a map configuration yourself."
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/configuration-driven-maps-framework/iframeable-configuration-map-example.md:3]:
```suggestion
description: "Describes the Iframeable configuration map with an example in HTML enabling you to try a map configuration yourself."
```

File: BingMaps/v8-web-control/map-control-concepts/configuration-driven-maps-framework/index.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Configuration Driven Maps framework | Microsoft Docs"
+description: "Describes the configuration driven maps framework and provides the configuration file format, examples, and additional references."
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/configuration-driven-maps-framework/index.md:3]:
```suggestion
description: "Describes the configuration driven maps framework and provides the configuration file format, examples, and related topics."
```

File: BingMaps/v8-web-control/map-control-concepts/contour-module-examples/index.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Contour Module Examples | Microsoft Docs"
+description: "Describes Contour module and provides the Basic Contour Layer and Contours from GeoJSON data examples and additional references."
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/contour-module-examples/index.md:3]:
```suggestion
description: "Describes Contour module and provides the Basic Contour Layer and Contours from GeoJSON data examples and links to related topics."
```

File: BingMaps/v8-web-control/map-control-concepts/custom-overlays/simple-static-overlay.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Simple Static Overlay | Microsoft Docs"
+description: "Describes the simple static overlay, provides sample code for the overlay, and outlines how to navigate the overlay."
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/custom-overlays/simple-static-overlay.md:3]:
```suggestion
description: "Describes the simple static overlay with an HTML example that creates a simple overlay consisting of 4 buttons that allow you to pan the map in different directions."
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by dereklh7** [comment]
@stevemunk I've implemented the requested changes. Please let me know if there are any issues preventing approval and merge.

**Pull request merged**